### PR TITLE
LPS-171347 search-experiences-service: Add search.empty.search attribute to searchContext to be possible return suggestions with 0 characters

### PR DESCRIPTION
**Pull request by gustavolimav**
https://issues.liferay.com/browse/LPS-171347

It was not possible to see the preview because `search.empty.search` attribute was not added in the search context, so I added it and it will not cause any problem because we have the threshold characters with 2 as the default value. Only if the user wants to change it to zero, the empty search in the suggestions will happen.

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:stop

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:reevaluate:1387401_5356

File: modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/resource/v1_0/SuggestionResourceImpl.java [hunk 1]
```diff
@@ -174,6 +174,7 @@ private SearchContext _createSearchContext(
 
 		SearchContext searchContext = new SearchContext();
 
+		searchContext.setAttribute("search.empty.search", Boolean.TRUE);
```
Comment (peerkar) [inline modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/resource/v1_0/SuggestionResourceImpl.java:177]: Can we use SearchContextAttributes constant here, too?

Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/resource/v1_0/SuggestionResourceImpl.java]: Ok

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:reevaluate:136null_2696

**Pull request closed without merging**